docs/middleware/healthcheck.md

```diff
@@ -4,7 +4,7 @@ id: healthcheck
 
 # Health Check
 
```

Comment:
Can you update v3 what's new docs as well

Reply:
@efectn I have added the information and migration changes in whats_new.md file